mount/mount_windows.go

```diff
@@ -84,7 +84,7 @@ func (mounter *Mounter) MountSensitive(source string, target string, fstype stri
 		// mount azure disk
 		bindSource = NormalizeWindowsPath(source)
 	} else {
-		if len(options) < 2 {
+		if len(sensitiveOptions) < 2 {
```

Comment:
/hold

options can be in options or sensitiveOptions... I would expect length checking to consider the total length of options+sensitiveOptions.